kategory-core/src/main/kotlin/kategory/data/Option.kt

```diff
@@ -195,6 +195,10 @@ fun <B> Option<B>.getOrElse(default: () -> B): B = fold({ default() }, { it })
  */
 fun <A, B : A> OptionKind<B>.orElse(alternative: () -> Option<B>): Option<B> = if (ev().isEmpty) alternative() else ev()
 
+fun <A : Any> A?.toOption(): Option<A> = Option.fromNullable(this)
+
+fun <A> Option<A>.toNative(): A? = this.getOrElse { null }
```

Comment:
I don't think we need a `toNative` as the clients can easily do the `getOrElse` themselves, no?

Reply:
@pakoito 👀

Reply:
I'm okay with the method, I'd rather just call it toNullable instead, but that's a nit that won't block the review.

Reply:
@pakoito not a nit, it'll be more consistent with `fromNullable()`!

Reply:
First @sockeqwe in the stars, now you checking out the PRs? Daaaamn it's been a great day :D

Reply:
Of course I check how accurately do you copy https://github.com/gojuno/koptional (I'm kidding, happy to see progress on KΛTEGORY)

@sockeqwe in stars, but @artem-zinnatullin in watchers! 😸 

// I only star projects I'm afraid to not find later, it's not the case with K∀∀∀∀TEGORY.

Reply:
:+1: toOption()
:-1: toNative()
should be `toNullable()`

Reply:
👀